src/test/java/api/CirculationRulesEngineAPITests.java

```diff
@@ -29,6 +31,11 @@ public CirculationRulesEngineAPITests() {
   }
 
   private void setRules(String rules) {
+    loanPoliciesFixture.create(getPolicyFromRule(rules, "l"));
+    noticePoliciesFixture.create(getPolicyFromRule(rules, "n"));
+    requestPoliciesFixture.allowAllRequestPolicy(getPolicyFromRule(rules, "r"));
+    overdueFinePoliciesFixture.create(getPolicyFromRule(rules, "o"));
+    lostItemFeePoliciesFixture.create(getPolicyFromRule(rules, "i"));
```

Comment:
Similarly to the other tests, I think creation of policies is better done explicitly separate from the rules being updated.